Restaurant.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import RestaurantModel from './RestaurantModel'
+
+class Restaurant extends React.Component {
+  constructor(props) {
+    super(props)
+    this.displayName = "Restaurant"
+  }
+
+  render() {
+    // helper for code simplicity.
+    var model = this.props.model
+
+    // Controller calculats food icon for HTML View presentation.
+    var foodTypeImage = "";
+    if (model.foodType === "Burger") {
+      foodTypeImage = "assets/img/Burger.png"
+    } else {
+      foodTypeImage = "http://placeponi.es/48/48"
+    }
+
+    // Prepare rating JSX
+    var ratings = [];
+    for (var i=0; i < model.rating; i++) {
+        ratings.push(<span className="glyphicon glyphicon-star" key={i}/>);
+    }
```

Comment:
lodash is awesome, it has a babel plugin that allows me to `import lodash` and have the plugin take only the chunks of the library that I'm actually using. I hope all 3rd party code in the world would behave in a similar way.

This is fixed by https://github.com/maximveksler/EatWell/commit/876050d03e9f54109467834e7fcffeaca17a680e and https://github.com/maximveksler/EatWell/commit/120cb1ae48afe9fd28ca5fa0db46034c43e40d2d